packages/core/upload/admin/src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -19,13 +20,21 @@ export const Breadcrumbs = ({ breadcrumbs, ...props }) => (
               .splice(index + 1, breadcrumbs.length - 1)
               .map(parent => parent.id)}
             key={`breadcrumb-${crumb?.id ?? 'menu'}`}
+            currentFolderId={currentFolderId}
+            onChangeFolder={onChangeFolder}
           />
         );
       }
 
-      if (crumb.href) {
+      if (crumb.href !== undefined) {
```

Comment:
What else could it be and would it be possible to elimnate this new falsy case?

Reply:
with other changes I made it could be null which is the root ML and needs to be also a CrumbLink